modules/acquia_cms_common/src/Services/ConfigImporterService.php

```diff
@@ -209,7 +210,7 @@ public function doImport(StorageComparer $storage_comparer, DrushLoggerManager $
         $message = 'The import failed due to the following reasons:' . "\n";
         $message .= implode("\n", $config_importer->getErrors());
 
-        watchdog_exception('config_import', $e);
+        Error::logException('config_import', $e);
```

Comment:
use DI here @rajeshreeputra

Reply:
as per discussion we cannot use DI here because of static method implementation in Error class similar to Crypt class